vendor/mcmc/mcmc.h

```diff
@@ -1,14 +1,16 @@
 #ifndef MCMC_HEADER
 #define MCMC_HEADER
 
+#include <sys/uio.h>
+#include <stdint.h>
+
 #define MCMC_OK 0
```

Comment:
nit: For readability, it might be useful to separate these out with some code comments?
(OK, ERR) -> Status

(NOT_CONN .. HAS_RESULT) -> Connection codes

(CODE_STORED .. CODE_END) -> Successful protocol response codes

(ERR_SHORT .. ERR_VALUE) -> Error protocol response codes

Reply:
they already kind of are, as you noted here CODE_* ERR_* etc. I deleted a few others that were unused and violated these principals already. Not sure it's worth splitting the rest out as they're entirely unused at the moment (see the commented out code)

Reply:
oh with comments. I guess so?